### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/tutorial/request-forms-and-files.md`

### PR DESCRIPTION
**Pull request by batlopes**
Added translation for pt to docs/pt/docs/tutorial/request-forms-and-files.md.

### CODE REVIEW

**Review by cassiobotaro — APPROVED** [review]
LGTM

File: docs/pt/docs/tutorial/request-forms-and-files.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+```Python hl_lines="8"
+{!../../../docs_src/request_forms_and_files/tutorial001.py!}
+```
+
+Os arquivos e campos de formulário serão carregados como dados de formulário e você receberá os arquivos e campos de formulário.
+
+E você pode declarar alguns dos arquivos como `bytes` e alguns como `UploadFile`.
+
+!!! warning "Aviso"
+    Você pode declarar vários parâmetros `File` e `Form` em uma *operação de caminho* , mas você também não pode declarar campos `Body` que você espera receber como JSON, pois a solicitação terá o corpo codificado usando `multipart/form-data` ao invés de `application/json`.
```
Comment (lsglucas) [inline docs/pt/docs/tutorial/request-forms-and-files.md:30]:
```suggestion
    Você pode declarar vários parâmetros `File` e `Form` em uma *operação de caminho*, mas não é possível declarar campos `Body` para receber como JSON, pois a solicitação terá o corpo codificado usando `multipart/form-data` ao invés de `application/json`.
```

I believe it sounds better this way

Comment (batlopes) [reply docs/pt/docs/tutorial/request-forms-and-files.md]: Done.

**Review by stlucasgarcia — APPROVED** [review]

**Review by peidrao — APPROVED** [review]

**Comment by tiangolo** [comment]
Great, thanks @batlopes! :cake: 

And thanks for the reviews @cassiobotaro, @peidrao, @lsglucas :cookie: :coffee:

**Pull request merged**